pkg/sql/sem/tree/col_types_test.go

```diff
@@ -90,3 +89,42 @@ func TestParseColumnType(t *testing.T) {
 		}
 	}
 }
+
+func TestParseColumnTypeAliases(t *testing.T) {
+	testData := []struct {
+		str          string
+		expectedStr  string
+		expectedType coltypes.T
+	}{
+		// FLOAT has always been FLOAT8
+		{"FLOAT", "CREATE TABLE a (b FLOAT8)", &coltypes.TFloat{Short: false}},
+		// A "naked" INT is 64 bits, for historical compatibility.
+		{"INT", "CREATE TABLE a (b INT8)", &coltypes.TInt{Width: 64}},
+		{"INTEGER", "CREATE TABLE a (b INT8)", &coltypes.TInt{Width: 64}},
+	}
+	for i, d := range testData {
+		sql := fmt.Sprintf("CREATE TABLE a (b %s)", d.str)
```

Comment:
I encourage subtests here. That will allow using t.Fatal instead of t.Error and removing the various continues.